app/com/arpnetworking/metrics/portal/reports/RecipientType.java

```diff
@@ -15,11 +15,14 @@
  */
 package com.arpnetworking.metrics.portal.reports;
 
+import com.arpnetworking.logback.annotations.Loggable;
+
 /**
  * Mechanisms by which reports can be sent to people.
  *
  * @author Spencer Pearson (spencerpearson at dropbox dot com)
  */
+@Loggable
```

Comment:
Can you confirm this actually does something? -- I'd love to see the before/after output for this. My suspicion is that it may be a no-op and that this is a bug in the logger factory.

Reply:
I only added this because I was overeager! Tested by hand -- turns out that enums serialize just fine into strings, as you can see up above.